Issue with the windows signing script

`npm run sign` gives me this error. For now, I am signing windows exe using the sign command directly.. 


`Error: Cannot find module '@/utils/logging'
Require stack:
- C:\2021\rancher-desktop\src\utils\childProcess.ts
- C:\2021\rancher-desktop\scripts\lib\sign-win32.ts
- C:\2021\rancher-desktop\scripts\sign.ts
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (C:\2021\rancher-desktop\src\utils\childProcess.ts:6:1)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Module.m._compile (C:\2021\rancher-desktop\node_modules\ts-node\src\index.ts:1295:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Object.require.extensions.<computed> [as .ts] (C:\2021\rancher-desktop\node_modules\ts-node\src\index.ts:1298:12)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (C:\2021\rancher-desktop\scripts\lib\sign-win32.ts:12:1)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Module.m._compile (C:\2021\rancher-desktop\node_modules\ts-node\src\index.ts:1295:23)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! rancher-desktop@0.6.1 sign: `node scripts/ts-wrapper.js scripts/sign.ts "C:\2021\rancher-desktop\dist\Rancher Desktop-0.6.1-win.zip"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the rancher-desktop@0.6.1 sign script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\GunasekharMatamalam\AppData\Roaming\npm-cache\_logs\2021-10-28T16_54_23_479Z-debug.log`

--- a/package.json
+++ b/package.json
@@ -111,6 +111,7 @@
     "spectron": "14.0.0",
     "ts-jest": "^24.3.0",
     "ts-node": "^10.0.0",
+    "tsconfig-paths": "^3.11.0",
     "vue-jest": "^3.0.7",
     "vue-template-compiler": "^2.6.12",
     "webpack": "^4.46.0",

--- a/scripts/lib/sign-win32.ts
+++ b/scripts/lib/sign-win32.ts
@@ -9,7 +9,7 @@ import { getSignVendorPath } from 'app-builder-lib/out/codeSign/windowsCodeSign'
 import yaml from 'js-yaml';
 import defaults from 'lodash/defaultsDeep';
 
-import * as childProcess from '../../src/utils/childProcess';
+import * as childProcess from '@/utils/childProcess';
 
 /**
  * Mandatory configuration for Windows.

--- a/scripts/ts-wrapper.js
+++ b/scripts/ts-wrapper.js
@@ -5,6 +5,10 @@
  * fine elsewhere, but not on the command line.
  */
 
+// Load tsconfig-paths so that ts-node can resolve files based on the 'paths'
+// compiler options key in tsconfig.json.
+require('tsconfig-paths/register');
+
 const { main: tsNodeMain } = require('ts-node/dist/bin');
 
 function main(args) {

